addons/editor/functions/fnc_handleSelectionChanged.sqf

```diff
@@ -0,0 +1,23 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Saves curator selected entities for use by modules.
+ *
+ * Arguments:
+ * 0: Curator <OBJECT>
+ * 0: Entity: group, marker, object, or waypoint <GROUP, STRING, OBJECT, ARRAY>
+ *
+ * Return Value:
+ * Handled <BOOL>
```

Comment:
Can this event handler be overridden?

Reply:
No, we don't store the index of the event anywhere, but we did the same for all other curator event handlers. We could consider bridging them with CBA event handlers, but I would make a separate PR for such a change.